### PR TITLE
Rearrange documentation organisation

### PR DESCRIPTION
**Pull request by pyhys**
Correct spelling errors

Update tutorial organisation

Use Github actions to build the documentation

### CODE REVIEW

**Review by olbjo — APPROVED** [review]
Looks good!
Some minor comments.
I have not rebuilt the documentation since the sphinx version is in conflict with 
another project. Maybe the document should also be built as part of the checks?

File: README.md [hunk 1]
```diff
@@ -1,32 +1,30 @@
-p-net Profinet device stack
-===========================
+# p-net Profinet device stack
```
[the remark this answers is not in the document]
Comment (pyhys) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
+- Documentation: [https://rt-labs.com/docs/p-net](https://rt-labs.com/docs/p-net)
+- Continuous integration: [https://github.com/rtlabs-com/p-net/actions](https://github.com/rtlabs-com/p-net/actions)
+- RT-Labs (stack integration, certification services and training): [https://rt-labs.com](https://rt-labs.com)
 
 [![Build Status](https://github.com/rtlabs-com/p-net/workflows/Build/badge.svg?branch=master)](https://github.com/rtlabs-com/p-net/actions?workflow=Build)
 [![CodeQL](https://github.com/rtlabs-com/p-net/workflows/CodeQL/badge.svg?branch=master)](https://github.com/rtlabs-com/p-net/actions?workflow=CodeQL)
 
-p-net
------
+## p-net
```
[the remark this answers is not in the document]
Comment (pyhys) [reply README.md]: Done

**Comment by olbjo** [comment]
Check out the formatting in the generated artifacts. For example the README.html.
I think the bulletlists used in earlier version is better.

**Comment by pyhys** [comment]
The bullet lists does not work when building on our setup on Github Actions. I need to investigate further.

**Comment by pyhys** [comment]
The missing bullets in the lists was due to a change in a dependency. Updated the version in the doc_requirements.txt file.

**Pull request merged**